thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/completeness/checker/DataCompletenessResource.java

```diff
@@ -0,0 +1,58 @@
+package com.linkedin.thirdeye.completeness.checker;
```

Comment:
Should we combine them or separate them?

If we separate them, then each one of them contains only the methods for either dashboard or workers. But it also makes sense to manage all data completeness related methods in one place.